packages/components/src/components/as-switch/as-switch.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { TestWindow } from '@stencil/core/dist/testing';
+import { Switch } from './as-switch';
+
+describe.only('as-switch', () => {
+  it('should build', () => {
+    expect(new Switch()).toBeTruthy();
+  });
+
+  describe('when the component is enabled', () => {
+    let element: HTMLAsToolbarElement;
+
+    beforeEach(async () => {
+      element = await _prepareTest('<as-switch id="switch"></as-switch>');
+    });
+
+    it('should emit an event with current status when component toggles', async () => {
+      const spy = _spyOnElement(element);
+
+      element.click();
+
+      expect(spy).toHaveBeenCalled();
+      expect(spy.mock.calls[0][0].detail).toBe(true);
+    });
+
+    it('should emit an event with current status when component toggles', async () => {
```

Comment:
This test is named same as the upper one.

I think you can initialize the test with the input checked and then check that it is unchecked correctly.

Actually, it is already checked in a test below those ones.